pages/_app.js

```diff
@@ -1,13 +1,29 @@
 import { useState } from "react";
 import GlobalStyle from "../styles";
-import { JobsDummyData } from "./api/db";
+import { JobsDummyData } from "./api/job_offers";
```

Comment:
```suggestion
import { jobOffersData } from "./api/job_offers";
```
Please rename this inside of `job_offers.js` as well.